libs/bulldozer/application/ui/dark-theme/src/lib/dark-theme.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { DarkThemeDirective } from './dark-theme.directive';
+import { DarkThemeService } from './dark-theme.service';
+
+
+@NgModule({
+  declarations: [DarkThemeDirective],
+  imports: [CommonModule],
+  providers: [DarkThemeService],
+  exports: [DarkThemeDirective],
+})
+export class HeavyDutyDarkThemeModule {}
```

Comment:
nitpick: Would you mind changing the name of the module from `HeavyDutyDarkThemeModule` to `DarkThemeModule`, since none of the modules have the "HeavyDuty" prefix?

Reply:
Of course, its ready.. thanks for pointing it out.